identity/aziot-identityd/src/lib.rs

```diff
@@ -371,7 +371,7 @@ impl Api {
         let _ = match trigger {
             ReprovisionTrigger::ConfigurationFileUpdate => {
                 self.id_manager
-                    .provision_device(self.settings.provisioning.clone(), true)
+                    .provision_device(self.settings.provisioning.clone(), false)
```

Comment:
In the case of DPS provisioning, this will lead to repeated re-provisioning for every file update, right? That is what we were talking about avoiding in the meeting. We need to check the backup to see if the provisioning info has changed or not.

I'll add you to the thread with Varun.

Reply:
Per that thread, I think we'll need to discuss further what the final behavior should be. For now, I'd like to fix the bug where config file update errors were getting ignored. Could you take another look at this PR just for that?